python/lsst/cp/pipe/ptc.py

```diff
@@ -393,7 +393,6 @@ def runDataRef(self, dataRef, visitPairs):
         # Produce coefficients for Polynomial ans Squared linearizers.
         dataset = self.fitPtcAndNonLinearity(dataset, self.config.ptcFitType,
                                              tableArray=lookupTableArray)
-
```

Comment:
I don't care, but others really seem to - please don't randomly delete lines for no reason.